src/components/navigation/document-tab-content.tsx

```diff
@@ -41,24 +41,24 @@ export const DocumentTabContent: React.FC<IProps> = ({ tabSpec }) => {
       : isProblemType(type) ? problem.title : document.getDisplayTitle(appConfig);
   };
 
-  function handleEditClick (document: DocumentModelType) {
+  function handleEditClick(document: DocumentModelType) {
     uiStore.problemWorkspace.setPrimaryDocument(document);
   }
 
-  const editButton = (type: string, sClass: string, document: DocumentModelType ) => {
+  const editButton = (type: string, sClass: string, document: DocumentModelType) => {
     return (
       (type === "my-work") || (type === "learningLog") ?
-        <div className={`edit-button ${sClass}`} onClick={()=>handleEditClick(document)}>
+        <div className={`edit-button ${sClass}`} onClick={() => handleEditClick(document)}>
```

Comment:
FYI, I like to line up the `?` with the `:`, so I would put the `?` on the next line.